drivers/misc/ethos_u/ethos_u.c

```diff
@@ -1,9 +1,10 @@
 /*
- * Copyright 2021-2022 Arm Limited and/or its affiliates <open-source-office@arm.com>
- *
+ * SPDX-FileCopyrightText: <text>Copyright 2021-2022, 2024 Arm Limited and/or its
+ * affiliates <open-source-office@arm.com></text>
```

Comment:
The `text` tag seems unconventional by Zephyr standards, but still passes CI so I'll let someone else decide if they feel this should be changed.

Reply:
After adding the "SPDX-FileCopyrightText" tag and current year, lines becomes longer than 100 chars and Zephyr's clang-format will complain about "ColumnLimit: 100" and breaks the line into two. Thus in order to have a properly formatted SPDX-FileCopyrightText field, the text needs to be enclosed in a <text>...</text> block